lib/kennel/filter.rb

```diff
@@ -45,7 +45,13 @@ def build_project_filter
     end
 
     def build_tracking_id_filter
-      (tracking_id = ENV["TRACKING_ID"]) && tracking_id.split(",").sort.uniq
+      (tracking_id = ENV["TRACKING_ID"]) && tracking_id.split(",").map { |id| tracking_id_path_conversion(id) }.sort.uniq
+    end
+
+    # allow users to paste the generated path of an objects to update it without manually converting
+    def tracking_id_path_conversion(tracking_id)
+      return tracking_id unless tracking_id.end_with?(".json")
+      tracking_id.split(/\/|\.json/).last(2).join(":")
```

Comment:
We could also do

```suggestion
      tracking_id.sub("generated/", "").sub(".json", "").sub("/", ":")
```

maybe. i.e. both the `generated/` and the `.json` are optional.

Reply:
yeah that reads better